docs/using.rst

```diff
@@ -412,33 +417,34 @@ Overriding the attribute protocol
 Subclasses which override the attribute-management methods provided by
 :class:`persistent.Persistent`, but must obey some constraints:
 
-:meth:`__getattribute__``
+
+``__getattribute__``
```

Comment:
I suspect probably what it was complaining about was that the :meth: appeared outside of a class body, and [without a leading class name](http://www.sphinx-doc.org/en/stable/domains.html#directive-py:class). So it didn't know what it was trying to describe.